client/state/ui/editor/image-editor/reducer.js

```diff
@@ -68,6 +69,13 @@ export function hasChanges( state = false, action ) {
 	return state;
 }
 
+export const originalAspectRatio = createReducer( null, {
+	[ IMAGE_EDITOR_IMAGE_HAS_LOADED ]: ( state, { width, height } ) => {
+		return { width, height };
+	},
```

Comment:
I avoided the one-liner here since it'd go over 80 characters:

<img width="671" alt="screen shot 2017-01-25 at 9 08 46 am" src="https://cloud.githubusercontent.com/assets/1270189/22300933/fb6ab4c8-e2dd-11e6-83be-ad186fdb7ef0.png">